src/Runners/PHPUnit/Options.php

```diff
@@ -183,7 +236,7 @@ protected static function vendorDir()
      * @param  array $options
      * @return array
      */
-    protected function filterOptions($options)
```

Comment:
now this is triple-redundant.

The description (I don't think it makes sense) says it returns a collection, then we `@return array` and have the typehint.

Either 
 * a useful description-text + typehints in the signature or
 * annotated param and return doc with a description like `@return array of default options` + typehinted signature or
 * only the signature including typehints or
 * as it was before (worst option probably)

Or does that make any style checker crying ? ;)